CRM/Core/PrevNextCache/Interface.php

```diff
@@ -130,4 +130,9 @@ public function getCount($cacheKey);
    */
   public function fetch($cacheKey, $offset, $rowCount);
 
+  /**
+   * Remove expired cache objects.
+   */
+  public function flush();
```

Comment:
In `CRM_Utils_Cache_Interface`, the `flush()` operation is one which proactively removes *all* items from the named cache. I generally perceive "flush" as an indiscriminate action.

It sounds like this operation is meant more as an incremental cleanup/timeout for expired items. For something that is incremental/targetted, I think `prune()` or `cleanup()` might be more evocative.